test/cypress/integration/web-payments/apple-pay.test.cy.js

```diff
@@ -326,6 +326,33 @@ describe('Apple Pay payment flow', () => {
       cy.get('#card-no').should('be.visible')
     })
 
+
+    it('Should not show Apple Pay when enabled but browser does not support it', () => {
```

Comment:
Not sure if my thinking is correct - can we add new tests to check for the new CSS classes?
Would be good to check each one to make sure it's def correct.